api-specs/Gateway-EE/3.5/kong-ee-3.5.yaml

```diff
@@ -328,6 +328,34 @@ components:
           tags:
             - silver-tier
           username: bob-the-builder
+    HMAC-auth-credential:
```

Comment:
These are enabled by the HMAC auth plugin - are you okay with adding them into these specs?

Reply:
Do you think this opens us up to requests for other plugin endpoints, if so I'll remove it and open a post 3.5 PR with more of the plugin endpoints that are unrelated to 3.5. I have a good way to add those in now.

Reply:
If you want to add the rest of the plugins, then go for it. I mostly wanted to check if this was appearing in here on purpose, and if it was, then we open the door for adding all the other plugin endpoints.